src/components/common/IconSVG/IconSVG.tsx

```diff
@@ -0,0 +1,27 @@
+import type { ImgHTMLAttributes } from 'react'
```

Comment:
This is a dumb component. I'd rather move it to /components/ui and wrap it using the `Icon` component from '@faststore/ui' so we don't need this `data-icon` attribute to style the icon, what do you thinnk? Also, it could be named of Icon, instead of IconSVG.